src/helpers/conversion.js

```diff
@@ -0,0 +1,56 @@
+const convertJSONtoPars = (srcValue) => {
+  const isArr = t => Array.isArray(t);
+  const isObj = o => typeof o === 'object';
+
+  const convertObject = (keymap) => {
+    const keystr = Object.keys(keymap).map((key) => {
+      const val = keymap[key];
+      if (isObj(val)) {
+        return `${key}=${convertObject(val)}`;
+      } else if (isArr(val)) {
+        // eslint-disable-next-line no-use-before-define
```

Comment:
please follow the eslint guideline

Reply:
@gidesan it's hard for me to follow that rule because `convertObject` and `convertArray` are both recursive functions that actually invokes each other.